src/api/hooks/useGetANS.ts

```diff
@@ -150,8 +152,13 @@ export const knownAddresses: Record<string, string> = {
   "0xa4e7455d27731ab857e9701b1e6ed72591132b909fe6e4fd99b66c1d6318d9e8":
     "Coinbase 2",
   "0x834d639b10d20dcb894728aa4b9b572b2ea2d97073b10eacb111f338b20ea5d7": "OKX 1",
+  "0x1e6009ce9d288f3d5031c06ca0b19a334214ead798a0cb38808485bd6d997a43": "OKX dapp",
```

Comment:
This is OKX Marketplace https://explorer.aptoslabs.com/account/0x1e6009ce9d288f3d5031c06ca0b19a334214ead798a0cb38808485bd6d997a43/modules/code/okx_auction?network=mainnet